bmwqemu.pm

```diff
@@ -192,6 +192,9 @@ our $backend;
 
 sub log_format_callback {
     my ($time, $level, @lines) = @_;
+    foreach ( @lines ) {
+        $_ =~ s/(?<!\A)^ */  /gm;
```

Comment:
the `$_ =~ ` part is default here, so it can be removed

Reply:
And maybe `@lines` could be renamed to something like `@items`, because `@lines` is misleading. each "line" can contain multiple lines.

Reply:
Ah, ok thanks